scalafmt-tests/src/test/scala/org/scalafmt/util/GitOpsTest.scala

```diff
@@ -1,12 +1,209 @@
 package org.scalafmt.util
 
 import java.io.File
+import java.nio.charset.StandardCharsets
+import java.nio.file.{Files, Paths}
 
-import org.scalatest.FunSuite
+import scala.util._
 
-class GitOpsTest extends FunSuite {
-  test("lsTree") {
-    val files = GitOps().lsTree(AbsoluteFile.userDir)
-    assert(files.exists(x => x.path.endsWith(s".gitignore")))
+import org.scalatest._
+import org.scalactic.source.Position
+
+class GitOpsTest extends fixture.FunSuite {
+
+  import Matchers._
+  import GitOpsTest._
+
+  val root = AbsoluteFile.userDir
+  val dirName = "gitTestDir"
+
+  override type FixtureParam = GitOpsImpl
+
+  // DESNOTE(2017-08-16, pjrt): Create a temporary git directory for each
+  // test.
+  override def withFixture(test: OneArgTest) = {
```

Comment:
Oh, one last thing. For native support, we may soon replace ScalaTest with thin FunSuite wrappers around utest. See https://github.com/scalameta/scalameta/issues/772 

It might be worth avoiding scalatest.fixture and Matchers to minimize the usage of scalatest features. We can deal with it once we port the tests to native, however.